SoftLayer/utils.py

```diff
@@ -371,6 +371,18 @@ def clean_time(sltime, in_format='%Y-%m-%dT%H:%M:%S%z', out_format='%Y-%m-%d %H:
         return sltime
 
 
+def verify_date(date):
```

Comment:
Is this function used anywhere? It would certainly be a good function to add to validate the date, I just don't see it used in this request.

Reply:
Yes, Its used in this line
https://github.com/softlayer/softlayer-python/pull/1830/files#diff-b5634f7308009391544d2cfdd7da670e1d499cd5c1c4d543bc486d4bc926ff5aR27